device_query/device_query.py

```diff
@@ -87,27 +88,29 @@ def main(verify):
             }
             encoded_post_data = bytes(json.dumps(post_data, indent=4), "utf-8")
 
-            time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-            print(f"[{time}] Sending request...")
+            logging.info(f"Sending request...")
 
-            if verify is not None:
-                r = requests.post(server_url, headers=headers, data=encoded_post_data, verify=verify)
+            if args.verify is not None:
+                r = requests.post(server_url, headers=headers, data=encoded_post_data, verify=args.verify)
             else:
                 r = requests.post(server_url, headers=headers, data=encoded_post_data, verify=False)
 
-            time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-            print(f"[{time}] Request returned {r.status_code} {r.reason}")
+            logging.info(f"Request returned {r.status_code} {r.reason}")
         except Exception as e:
-            time = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-            print(f"[{time}] Error: {e}")
-            pass
+            logging.error(f"Error: {e}")
         finally:
             sleep(update_interval)
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--verify", help="path to the certificate file that should be used to verify requests")
+    parser.add_argument("-v", "--verbose", action="store_true", help="Shows additional log messages")
     args = parser.parse_args()
 
-    main(args.verify)
+    if args.verbose:
+        logging.basicConfig(format="[%(asctime)s] %(message)s", level=logging.DEBUG)
+    else:
+        logging.basicConfig(format="[%(asctime)s] %(message)s")
```

Comment:
does this setting incur that we do not log the `info` log commands in normal mode?

Reply:
Yes

Reply:
good!